docs/design.md

```diff
@@ -317,17 +322,16 @@ There are several scenarios in which transactions interact:
   transaction will then notice its timestamp has been pushed, and
   restart). If it has the lower priority, it retries itself using as
```

Comment:
the word "it" in this sentence and the preceding one is ambiguous since the subject of the last sentence to have an explicit subject is the conflict. can you disambiguate while you're here?